### PR TITLE
Lets try this again!

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: conda_forge_tick/migrators.xsh [hunk 1]
```diff
@@ -1345,6 +1346,7 @@ class MigrationYaml(Migrator):
             self.pr_limit = 2
         elif number_pred < 7:
             self.pr_limit = 5
+        self.bump_number = yaml_contents.get("build_number", 0)
```
Comment (CJ-Wright) [inline conda_forge_tick/migrators.xsh]:
```suggestion
        self.bump_number = yaml_contents.get("build_number", 1)
```
Most migrations need a rebuild, only matrix expansions don't.

**Comment by CJ-Wright** [comment]
@regro/auto-tick I think this is ready for review. We now have a test for this functionality.
Thank you @soapy1 for this the first time around!

**Comment by scopatz** [comment]
Thanks @CJ-Wright!

**Comment by CJ-Wright** [comment]
We need to revert this again

**Comment by mariusvniekerk** [comment]
This test doesn't test the right thing

**Comment by CJ-Wright** [comment]
That doesn't really have enough context for me to go off of.

File: conda_forge_tick/auto_tick.xsh [hunk 2]
```diff
@@ -519,15 +518,16 @@ def migration_factory(migrators, gx, pr_limit=50):
...
-        obj_version = loaded_yaml.get('__migrator', {}).get('migration_number', 0)
-        exclude_packages = set(loaded_yaml.get('__migrator', {}).get('exclude', []))
-        bump_number = loaded_yaml.get('__migrator', {}).get('bump_number', 1)
-        package_names = ((set(loaded_yaml)|set(l.replace('_', '-') for l in loaded_yaml)) & set(gx.nodes)) - exclude_packages
         print(os.path.splitext(yaml_file)[0])
-        add_rebuild_migration_yaml(migrators, gx, package_names, yaml_contents,
-                                   migration_name=os.path.splitext(yaml_file)[0],
-                                   pr_limit=pr_limit,
-                                   obj_version=obj_version, bump_number=bump_number)
+        yaml_config = loaded_yaml.get('__migrator', {})
```
Comment (mariusvniekerk) [inline conda_forge_tick/auto_tick.xsh]: This is the problem

Comment (CJ-Wright) [reply conda_forge_tick/auto_tick.xsh]: Thank you!

**Pull request merged**